jdaviz/configs/specviz/specviz.ipynb

```diff
@@ -0,0 +1,66 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
```

Comment:
Why does this notebook have outputs?  Or more broadly... does it even need to exist at all? I know it's in cubeviz, but it's not clear to me what purpose it serves - if not needed perhaps it should be moved to the notebook examples directory?

Reply:
I used this notebook to verify all the features worked, and before I threw it out, I thought this might be a useful resource to eventually show how Speviz works and, more broadly, how the helper classes work. But if this isn't necessary, I could also see just getting rid of it

Reply:
It's not entirely clear to me either. These config notebooks are necessary for everything to load, but what exactly it does is a little ambiguous to me. When you load the web version, you get a brief message of "Executing cells 1 of 2" before it loads. Those cells are presumably these

Reply:
Ohhh, gotcha.  @nmearl can confirm but I'm guessing it's that this is the notebook that voila is executing to show that interface.  In that case I think you do want to clear the output since it's executing anyway (not critical, but it'll be a bit less confusing for people who stumble upon this I think)

Reply:
Ok, I've also cleared the output from the cubeviz notebook as well